docs/deploy-options.md

```diff
@@ -1,44 +1,37 @@
-# Deployment options
+# Deployment and testing
 
-The [README][0] shows you a simple way to get started with Contour on your cluster. This topic explains the details and shows you additional options.  Most of this covers running Contour using a Kubernetes Service of `Type: LoadBalancer`. If you don't have a cluster with that capability or if you don't want to use it see the [Running without a Kubernetes LoadBalancer](#running-without-a-kubernetes-loadbalancer) section.
+The [README](../README.md#get-started) shows you a simple way to get started with Contour on your cluster.
+This topic explains the details and shows you additional options.
+Most of this covers running Contour using a Kubernetes Service of `Type: LoadBalancer`.
+If you don't have a cluster with that capability or if you don't want to use it see the [Running without a Kubernetes LoadBalancer](#running-without-a-kubernetes-loadbalancer) section.
 
-## Deployment or DaemonSet?
+## Installation
 
-We provide example deployment manifests for setting up Contour by creating either a DaemonSet or a Deployment.
+### Recommended installation details
 
-- The DaemonSet creates an instance of Contour that runs on each node in your cluster.
-- The Deployment creates two instances of Contour that run on the cluster, on two arbitrary nodes.
+The recommended installation of Contour has Contour running in a Deployment and Envoy in a Daemonset, with TLS securing the xDS gRPC communication between them.
+The `contour` example (which the README points you to) will install this for you.
```

Comment:
which readme (and why isn't it a hyperlink), maybe making `contour` clickable will do.